src/languages/index.js

```diff
@@ -4,4 +4,9 @@ module.exports = [
         key: "en",
         table: require("./en"),
     },
+    {
+        name: "Nederlands",
```

Comment:
Shouldn't this be "Dutch"?

Reply:
Not in my opinion. Dutch is Dutch is Nedelands. So if someone couldn’t read English, he could choose the right language.